src/renderer.ts

```diff
@@ -492,226 +752,148 @@ const render = (format: Format, excludeStyles = false) => (
 			return h(CalloutForm, { campaign, format, description });
 		}
 
-		case ElementKind.Embed: {
-			const props = {
-				dangerouslySetInnerHTML: {
-					__html: element.html,
-				},
-			};
-
-			const figureCss = css`
-				margin: ${remSpace[4]} 0;
-				${darkModeCss`
-                    background: white;
-                    padding: ${remSpace[2]};
-                `}
-			`;
-			const captionStyles = css`
-				${textSans.xsmall()}
-				color: ${textColour.supporting};
-			`;
+		case ElementKind.Embed:
+			return embedRenderer(element);
 
-			const figure = (alt: string | null): ReactElement => {
-				const children = [
-					h('div', props),
-					styledH('figcaption', { css: captionStyles }, alt),
-				];
-				return styledH('figure', { css: figureCss }, children);
-			};
-
-			return pipe2(
-				element.alt,
-				map((alt) => figure(alt)),
-				withDefault(figure(null)),
-			);
+		case ElementKind.Instagram:
+			return instagramRenderer(element);
+
+		case ElementKind.ExplainerAtom: {
+			return h(ExplainerAtom, { ...element });
 		}
 
-		case ElementKind.Instagram: {
-			const props = {
-				dangerouslySetInnerHTML: {
-					__html: element.html,
-				},
-			};
+		case ElementKind.GuideAtom:
+			return guideAtomRenderer(format, element);
+
+		case ElementKind.QandaAtom:
+			return qandaAtomRenderer(format, element);
 
-			return h('div', props);
+		case ElementKind.ProfileAtom:
+			return profileAtomRenderer(format, element);
+
+		case ElementKind.TimelineAtom:
+			return timelineAtomRenderer(format, element);
+
+		case ElementKind.ChartAtom: {
+			return h(ChartAtom, { ...element });
 		}
 
-		case ElementKind.ExplainerAtom: {
-			return h(ExplainerAtom, { ...element });
+		case ElementKind.InteractiveAtom:
+			return interactiveAtomRenderer(format, element);
+
+		case ElementKind.MediaAtom:
+			return mediaAtomRenderer(format, element);
+
+		case ElementKind.AudioAtom:
+			return audioAtomRenderer(format, element);
+
+		case ElementKind.QuizAtom:
+			return quizAtomRenderer(format, element);
+	}
+};
+
+const renderEditions = (format: Format, excludeStyles = false) => (
+	element: BodyElement,
+	key: number,
+): ReactNode => {
+	switch (element.kind) {
+		case ElementKind.Text:
+			return textRenderer(format, excludeStyles, element);
+
+		case ElementKind.Image:
+			return imageRenderer(format, element, key);
+
+		case ElementKind.Pullquote: {
+			const { quote, attribution } = element;
+			return h(Pullquote, { quote, attribution, format, key });
```

Comment:
Is the next step to build an `EditionsPullquote` component?

Reply:
Indeed 👍